.github/codeql.yml

```diff
@@ -0,0 +1,62 @@
+name: "Code scanning - action"
+
+on:
+  pull_request:
+    types:
+    - opened
+    - reopened
+    - synchronize
+    - ready_for_review
+  push:
+    # run workflow when merging to main/master or develop
+    branches:
+      - main
+      - master
+      - develop
+  schedule:
+    - cron: '0 19 * * 0'
+
+jobs:
+  CodeQL-Build:
+
+    # CodeQL runs on ubuntu-latest and windows-latest
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
```

Comment:
I assume you copied it from the old auth2 repo. but do we need to upgrade actions/checkout@v2 here?

Reply:
👍